.env.production

```
@@ -1,4 +1,5 @@
 REACT_APP_DOPPLER_LEGACY_URL=https://app2.fromdoppler.com
+REACT_APP_DOPPLER_SITES_URL= https://app2.fromdoppler.com
```

If I am not wrong, this is not the right URL, could you ask @hernangalvan ?

Also, what do you think about using the same URL in all environments?

---

[Confirmed](https://makingsense.slack.com/archives/GKPSZJC4U/p1560973966004000): 

```suggestion
REACT_APP_DOPPLER_SITES_URL= https://www.fromdoppler.com
```